packages/mediapipe-core/ffigen.yaml

```diff
@@ -0,0 +1,13 @@
+name: "MediaPipeCommonBindings"
+description: "Bindings for shared MediaPipe structs common across many tasks"
+output:
+  bindings: "lib/src/mediapipe_common_bindings.dart"
+  symbol-file:
+    output: "package:mediapipe_core/generated/core_symbols.yaml"
+    import-path: "package:mediapipe_core/src/mediapipe_common_bindings.dart"
+headers:
+  entry-points:
+    - "third_party/mediapipe/tasks/c/core/base_options.h"
```

Comment:
If you're only going to commit the header files to the repo that are in this list, consider making this list `- "third_party/mediapipe/tasks/c/*"`. We should have glob matching.